packages/api/tests/models/test-pdrs-model.js

```diff
@@ -56,6 +56,35 @@ const createPdrMessage = ({
   }
 });
 
+test('generatePdrRecord() returns null if message.payload.pdr is not set', (t) => {
+  const pdrRecord = Pdr.generatePdrRecord({});
+
+  t.is(pdrRecord, null);
+});
+
+test('generatePdrRecord() returns null if message.payload.pdr.name is not set', (t) => {
+  const pdrRecord = Pdr.generatePdrRecord({
+    payload: {
+      pdr: {}
+    }
+  });
+
+  t.is(pdrRecord, null);
+});
+
+// Addresses CUMULUS-1644
```

Comment:
Do we need to keep around the ticket number? I thought we usually only do that to remind ourselves of something that hasn't yet been addressed. Not particularly important though

Reply:
It's a little weird when you have a situation where something was happening before, and then you remove that functionality. Do you test to verify that you've properly _removed_ the functionality? I didn't want someone to come across this test later and say, "why are you testing that something _doesn't_ happen?" The two options that make sense to me are 1) remove the test entirely or 2) leave the test, but explain (with the ticket number) _why_ we're doing this test.

Reply:
Fixed in 87f315495